### ISSUE TITLE
errata import missing perl dependencies

### ISSUE THREAD

**Issue by MaxHerrmannSVA** [stdevel/ansible-uyuni#41]
Since opensuse 15.4 the perl TimeDate module is not installed anymore and also is not available on the default repositories.
To install the missing package we have to add the dedicated perl repo first, then we can install it:

```console
# zypper addrepo https://download.opensuse.org/repositories/devel:languages:perl/15.4/devel:languages:perl.repo
# zypper refresh
# zypper in perl-TimeDate
```

After that the import will run again

**Comment by stdevel** [comment]
This should be fixed in branch `bug-perl-tiemdate-41` - can you try it @MaxHerrmannSVA?

**Comment by stdevel** [comment]
Any news, @MaxHerrmannSVA? :)

**Comment by MaxHerrmannSVA** [comment]
Yeah, thanks for reminding me.
It works like a charm now! Thanks ;)

**Comment by stdevel** [comment]
Fixed in commit 0c4096753ea4602789259320c57f7d7c2141f763.
